rcl/test/resources/test_arguments/test_parameters.2.yaml

```diff
@@ -0,0 +1,5 @@
+another_node:
```

Comment:
It would be good to test two files with overlapping parameter overrides.

Reply:
Done in https://github.com/ros2/rcl/pull/508/commits/c1f4c3ce1dc69cf0d29cf1c72873d582efa7a557